tests/main/security-device-cgroups-classic/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Check that device nodes are available in classic
+
+details: |
+    This tests that a framebuffer device is accessible in classic and makes
+    sure that other devices are still accessible (ie, the cgroup is not in
+    effect).
+
+systems: [-fedora-*, -opensuse-*, -ubuntu-core-*]
+
+prepare: |
+    # Create framebuffer device node and give it some content we can verify
+    # the test snap can read.
+    if [ ! -e /dev/fb255 ]; then
+        mknod /dev/fb255 c 29 255
+        touch /dev/fb255.spread
+    fi
+
+    echo "Given a snap declaring a plug on framebuffer is installed in classic"
+    . $TESTSLIB/snaps.sh
+    install_local_classic test-classic-cgroup
+
+restore: |
+    if [ -e /dev/fb255.spread ]; then
```

Comment:
I think we can just simplify this to:
```
rm -f /dev/fb255.spread
```

Reply:
The idea is that we mock up the devices if they aren't there. If we created them, we clean what we mocked. This is the same technique we used in security-device-cgroups for the nvidia devices.

Reply:
My point is that the code is equivalent to `rm -f /dev/fb255.spread` since if the device is not there `-f` will just do nothing. Am I missing anything?

Reply:
Ah, wait. I see this now, sorry. I don't know how I missed this.

Reply:
Right, it is a stamp file to let us know to remove the device or not.